sql/handler.cc

```diff
@@ -7743,6 +7814,9 @@ int handler::ha_write_row(uchar *buf)
   if (unlikely(error= binlog_log_row(table, 0, buf, log_func)))
     DBUG_RETURN(error); /* purecov: inspected */
 
+  if (!is_fake_change_enabled(ha_thd()))
```

Comment:
Thanks, I considered that but then if someone is using fake changes (unlikely, heh), they will have 100% branch prediction miss so didn't think it was a good case for it.